src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props

```diff
@@ -78,6 +78,20 @@ Copyright (c) .NET Foundation. All rights reserved.
     <PackageRequireLicenseAcceptance Condition="'$(PackageRequireLicenseAcceptance)'==''">false</PackageRequireLicenseAcceptance>
   </PropertyGroup>
 
+  <!-- Automatically reference NETStandard.Library or Microsoft.NETCore.App package if targeting the corresponding target framework.
+      We can refer here in the .props file to properties set in the .targets files because items and their conditions are
+      evaluated in the second pass of evaluation, after all properties have been evaluated. -->
+  <ItemGroup Condition=" '$(DisableImplicitFrameworkReferences)' != 'true' and '$(TargetFrameworkIdentifier)' == '.NETStandard'">
+    <PackageReference Include="NETStandard.Library" Version="$(NetStandardDefaultPackageVersion)"/>
```

Comment:
No.  I haven't yet written up the thinking behind this change, but basically since right now the Microsoft.NETCore.App version needs to match the TargetFrameworkVersion, the idea is to hide the package reference entirely.  The NETStandard.Library package supports lower versions of the target framework, so it's OK to surface updates to that package in the UI.